src/main/java/data/Receipt.java

```diff
@@ -0,0 +1,46 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import com.google.appengine.api.blobstore.BlobKey;
+import java.util.Set;
+
+/* Class to represent a receipt and its properties.*/
+public class Receipt {
+  private long id;
+  private long userId;
+  private long timestamp;
+  private BlobKey blobkey;
+  private String imageURL;
```

Comment:
The Java style guide defines camel case disregarding the casing of the original word, so in camel case this would be imageUrl.

https://google.github.io/styleguide/javaguide.html#s5.3-camel-case